### PR TITLE
Align the version of jdt.core in maven to the one from gradle

### PR DESCRIPTION
**Pull request by holgerschill**
Signed-off-by: Holger Schill <Holger.Schill@itemis.de>

### CODE REVIEW

File: releng/org.eclipse.xtend.tycho.parent/pom.xml [hunk 1]
```diff
@@ -24,8 +24,8 @@
 		<log4j.version>1.2.16</log4j.version>
 		<commons-logging.version>1.1.3</commons-logging.version>
 		<commons-cli.version>1.2</commons-cli.version>
-		<equinox-common-version>3.6.0</equinox-common-version>
-		<jdt-core-version>3.10.0</jdt-core-version>
+		<equinox-common-version>3.8.0</equinox-common-version>
+		<jdt-core-version>3.12.2</jdt-core-version>
```
Comment (spoenemann) [inline releng/org.eclipse.xtend.tycho.parent/pom.xml]: See my comment on eclipse/xtext-eclipse#152

Comment (holgerschill) [inline releng/org.eclipse.xtend.tycho.parent/pom.xml]: Even when the properties are not used we could merge that, right?

Comment (spoenemann) [inline releng/org.eclipse.xtend.tycho.parent/pom.xml]: Yes that wouldn't break the build, but it would be better to remove unused properties.

Comment (holgerschill) [inline releng/org.eclipse.xtend.tycho.parent/pom.xml]: I'll remove them later. Would be good to have the gradle stuff in the repo.

Comment (spoenemann) [inline releng/org.eclipse.xtend.tycho.parent/pom.xml]: 👍

**Pull request closed without merging**